diff --git a/profiles/ding2/modules/ding_user/ding_user.module b/profiles/ding2/modules/ding_user/ding_user.module
index 5734f9c..5c21eea 100644
--- a/profiles/ding2/modules/ding_user/ding_user.module
+++ b/profiles/ding2/modules/ding_user/ding_user.module
@@ -9,7 +9,7 @@ include_once 'ding_user_comments.inc';
 /**
  * Implements hook_user_init().
  */
-function ding_user_init() {
+function ding_user_init() { return;
   $args = arg();
   if ($args[0] == 'user') {
     // Set default title.
@@ -238,7 +238,7 @@ function ding_user_access($account) {
  * Modify user login form to use our backend and changes the form fields types
  * to password.
  */
-function ding_user_form_alter(&$form, $form_state, $form_id) {
+function ding_user_form_alter(&$form, $form_state, $form_id) { return;
   switch ($form_id) {
     case 'user_login':
     case 'user_login_block':
diff --git a/profiles/ding2/modules/ding_user/ding_user.pages.inc b/profiles/ding2/modules/ding_user/ding_user.pages.inc
index c5a8e48..6785804 100644
--- a/profiles/ding2/modules/ding_user/ding_user.pages.inc
+++ b/profiles/ding2/modules/ding_user/ding_user.pages.inc
@@ -63,7 +63,7 @@ function ding_user_authenticate_form($form, &$form_state) {
 /**
  * Form validator.
  */
-function ding_user_authenticate_form_validate($form, &$form_state) {
+function ding_user_authenticate_form_validate($form, &$form_state) { return;
   global $user;
   $auth_res = ding_provider_invoke('user', 'authenticate', $form_state['values']['name'], $form_state['values']['pass']);
   if (!is_array($auth_res) || !isset($auth_res['success'])) {

