diff --git a/sites/all/modules/reol_lazy_load_images/reol_lazy_load_images.module b/sites/all/modules/reol_lazy_load_images/reol_lazy_load_images.module
index 146b55579..179b829e4 100644
--- a/sites/all/modules/reol_lazy_load_images/reol_lazy_load_images.module
+++ b/sites/all/modules/reol_lazy_load_images/reol_lazy_load_images.module
@@ -152,7 +152,7 @@ function theme_reol_base_image($variables) {
   }
 
   // @see https://act-rules.github.io/rules/c487ae
-  $label = $attributes['title'] ?? $attributes['alt'] ?? NULL;
+  $label = isset($attributes['title']) ? $attributes['title'] : (isset($attributes['alt']) ? $attributes['alt'] : NULL);
   if (NULL !== $label) {
     $attributes['aria-label'] = $label;
   }
